sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/utils/ShutdownHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.azure.mobile.utils;
+
+import android.os.Process;
+import android.support.annotation.VisibleForTesting;
+
+/**
+ * Shutdown helper.
+ */
+public class ShutdownHelper {
```

Comment:
Ignoring handlers and shutdown are not really in use in Mobile Center. We inherited part of this code from Hockey App and we should take this opportunity to get rid of it as it was private API to ignore handler in Mobile Center.